projects/fsastorefront-e2e-cypress/cypress/helpers/checkout/insurance/auto.ts

```diff
@@ -39,6 +39,70 @@ export function populateAutoInformation() {
   });
 }
 
+export function populateAutoMonthlyAudi() {
```

Comment:
Just a note, those 4 methods can be refactored in future as one that takes  6 arguments. 

- vehicleMake
- vehicleModel
...
- vehicleValue

since rest of the inputs are same for all cases :)

Reply:
will be covered as part of https://cxjira.sap.com/browse/FSA-4938